sdk/python/test/e2e/test_e2e_mpijob.py

```diff
@@ -54,101 +59,101 @@ def test_sdk_e2e_with_gang_scheduling(job_namespace):
         replicas=1,
         restart_policy="Never",
         template=V1PodTemplateSpec(
-            metadata=V1ObjectMeta(annotations={constants.ISTIO_SIDECAR_INJECTION: "false"}),
+            metadata=V1ObjectMeta(
+                annotations={constants.ISTIO_SIDECAR_INJECTION: "false"}
+            ),
             spec=V1PodSpec(
                 containers=[launcher_container],
                 scheduler_name=get_pod_spec_scheduler_name(GANG_SCHEDULER_NAME),
-            )
+            ),
         ),
     )
 
     worker = KubeflowOrgV1ReplicaSpec(
         replicas=1,
         restart_policy="Never",
         template=V1PodTemplateSpec(
-            metadata=V1ObjectMeta(annotations={constants.ISTIO_SIDECAR_INJECTION: "false"}),
+            metadata=V1ObjectMeta(
+                annotations={constants.ISTIO_SIDECAR_INJECTION: "false"}
+            ),
             spec=V1PodSpec(
                 containers=[worker_container],
                 scheduler_name=get_pod_spec_scheduler_name(GANG_SCHEDULER_NAME),
-            )
+            ),
         ),
     )
 
-    mpijob = generate_mpijob(launcher, worker, KubeflowOrgV1SchedulingPolicy(min_available=10), job_namespace)
-    patched_mpijob = generate_mpijob(launcher, worker, KubeflowOrgV1SchedulingPolicy(min_available=2), job_namespace)
+    mpijob = generate_mpijob(
+        job_namespace, launcher, worker, KubeflowOrgV1SchedulingPolicy(min_available=10)
+    )
+    patched_mpijob = generate_mpijob(
+        job_namespace, launcher, worker, KubeflowOrgV1SchedulingPolicy(min_available=2)
+    )
 
-    TRAINING_CLIENT.create_mpijob(mpijob, job_namespace)
-    logging.info(f"List of created {constants.MPIJOB_KIND}s")
-    logging.info(TRAINING_CLIENT.list_mpijobs(job_namespace))
+    TRAINING_CLIENT.create_job(job=mpijob, namespace=job_namespace)
+    logging.info(f"List of created {TRAINING_CLIENT.job_kind}s")
+    logging.info(TRAINING_CLIENT.list_jobs(job_namespace))
 
-    verify_unschedulable_job_e2e(
-        TRAINING_CLIENT,
-        JOB_NAME,
-        job_namespace,
-        constants.MPIJOB_KIND,
-    )
+    verify_unschedulable_job_e2e(TRAINING_CLIENT, JOB_NAME, job_namespace)
 
-    TRAINING_CLIENT.patch_mpijob(patched_mpijob, JOB_NAME, job_namespace)
-    logging.info(f"List of patched {constants.MPIJOB_KIND}s")
-    logging.info(TRAINING_CLIENT.list_mpijobs(job_namespace))
+    TRAINING_CLIENT.update_job(patched_mpijob, JOB_NAME, job_namespace)
+    logging.info(f"List of patched {TRAINING_CLIENT.job_kind}s")
```

Comment:
```suggestion
    logging.info(f"List of updated {TRAINING_CLIENT.job_kind}s")
```

`updated` might be proper.

Reply:
Updated these log line.